### PR TITLE
ir/config: assign default to missing port in TCP addresses

### LINKED ISSUE #2420: missing port in address
This:
```Yaml
morph:
  seed_nodes:
    - hostA
    - hostB
    - hostC
```

gives an error:
```
invalid blockchain configuration: invalid seed nodes 'morph.consensus.seed_nodes' (TCP addresses): address hostA: missing port in address
```

1. assign a port
2. use it by default

### PR DESCRIPTION
**Pull request by End-rey**
Closes #2420.

Is it necessary to document default ports, except the example config?

### CODE REVIEW

**Review by carpawell — APPROVED** [review]
> Is it necessary to document default ports, except the example config?

I think I would be great but we do not have any doc file for IR app for now.

File: pkg/innerring/config.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ const (
 	cfgPathFSChainRPCEndpoints   = cfgPathFSChain + ".endpoints"
 	cfgPathFSChainLocalConsensus = cfgPathFSChain + ".consensus"
 	cfgPathFSChainValidators     = cfgPathFSChain + ".validators"
+	p2pDefaultAddressPort        = ":20333"
```
Comment (cthulhu-rider) [inline pkg/innerring/config.go:28]: lets create a new `const` block, this one is already designated for the paths

Comment (cthulhu-rider) [inline pkg/innerring/config.go:28]: i'd name them `*DefaultListenPort` to emphasize the purpose

Comment (cthulhu-rider) [inline pkg/innerring/config.go:28]: its strange to see a colon in a port constant, dont u think?

File: pkg/innerring/config.go [hunk 2]
```diff
@@ -372,12 +374,15 @@ func parseConfigPublicKeys(v *viper.Viper, key, desc string) (keys.PublicKeys, e
...
 
-func parseConfigAddressesTCP(v *viper.Viper, key, desc string) ([]string, error) {
+func parseConfigAddressesTCP(v *viper.Viper, key, desc string, defaultPort string) ([]string, error) {
 	ss, err := parseConfigStrings(v, key, desc)
 	if err != nil {
 		return nil, err
 	}
 	for i := range ss {
+		if !strings.Contains(ss[i], ":") {
+			ss[i] = ss[i] + defaultPort
```
Comment (cthulhu-rider) [inline pkg/innerring/config.go:384]:
```suggestion
			ss[i] += defaultPort
```

**Review by cthulhu-rider — COMMENTED** [review]
correct overall

**Review by roman-khimov — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**